plugin-src/transformers/partials/transformChildren.ts

```diff
@@ -9,8 +8,6 @@ export const transformChildren = async (
   baseY: number = 0
 ): Promise<Children> => {
   return {
-    children: (
-      await Promise.all(node.children.map(child => transformSceneNode(child, baseX, baseY)))
-    ).filter((child): child is PenpotNode => !!child)
+    children: await transformMaskChildren(node.children, baseX, baseY)
```

Comment:
It sounds strange to make all children pass through this mask children, maybe only some specific elements can have masked children? If so, I would suggest to create an alternative version of this function for just that purpose.